dm/syncer/data_validator.go

```diff
@@ -725,10 +725,10 @@ func (v *DataValidator) Stop() {
 
 func (v *DataValidator) stopInner() {
 	v.Lock()
-	defer v.Unlock()
```

Comment:
i was thinking whether we could change the lock in `error process routine` to `state lock`. ok for now.